doc/releases/reef.rst

```diff
@@ -0,0 +1,219 @@
+    to prioritize client I/O or the ``high_recovery_ops`` profile to prioritize
+    recovery I/O.
+  * QoS parameters including ``reservation`` and ``limit`` are now specified in 
+    terms of a fraction (range: 0.0 to 1.0) of the OSD's IOPS capacity.
+  * The cost parameters (``osd_mclock_cost_per_io_usec_*`` and
+    ``osd_mclock_cost_per_byte_usec_*``) have been removed. The cost of an 
+    operation is now a function of the random IOPS and maximum sequential
+    bandwidth capability of the OSD's underlying device.
+  * Degraded object recovery is given higher priority than misplaced
+    object recovery because degraded objects present a data safety issue that
+    is not present with objects that are merely misplaced. As a result,
+    backfilling operations with the ``balanced`` and ``high_client_ops`` mClock
+    profiles might progress more slowly than in the past, when backfilling
+    operations used the 'WeightedPriorityQueue' (WPQ) scheduler.
+  * The QoS allocations in all the mClock profiles are optimized in 
+    accordance with the above fixes and enhancements.
+  * For more details, see:
+    https://docs.ceph.com/en/latest/rados/configuration/mclock-config-ref/
+* The ``active_clients`` array displayed by the ``ceph mgr dump`` command now
+  has a ``name`` field that shows the name of the manager module that
+  registered a RADOS client. Previously, the ``active_clients`` array showed
+  the address of a module's RADOS client, but not the name of the module.
+
+RBD
+~~~
+
+* RBD: The semantics of compare-and-write C++ API (`Image::compare_and_write`
+  and `Image::aio_compare_and_write` methods) now match those of the C API.
+  Both the compare and the write steps operate only on ``len`` bytes, even if
+  their respective buffers are larger. The previous behavior that compared only
+  up to the size of the compare buffer was prone to subtle breakage when
+  straddling a stripe unit boundary.
+* RBD: The ``compare-and-write`` operation is no longer limited to 512-byte 
+  sectors. Assuming proper alignment, it now allows operating on stripe units
+  (4M by default).
+* RBD: There is a new ``rbd_aio_compare_and_writev`` API method that supports 
+  scatter/gather on compare buffers as well as on write buffers. This
+  complements the existing ``rbd_aio_readv`` and ``rbd_aio_writev`` methods.
+* RBD: The ``rbd device unmap`` command has gained a ``--namespace`` option.
+  Support for namespaces was added to RBD in Nautilus 14.2.0. It has been
+  possible to map and unmap images in namespaces using the `image-spec` syntax
+  since 14.2.0, but an option like this (similar to options available in most
+  other commands) was missing.
+* RBD: All rbd-mirror daemon perf counters became labeled and as such are now
+  emitted only by the new ``counter dump`` and ``counter schema`` commands. As
+  part of this conversion, many also got renamed to better disambiguate
+  journal-based and snapshot-based mirroring.
```

Comment:
@zdover23 - Edited/reviewed version from 2023 June 18: RBD: All rbd-mirror daemon perf counters have become labeled and are now emitted only by the new ``counter dump`` and ``counter schema`` commands. As part of the conversion, many were also renamed in order to better disambiguate journal-based and snapshot-based mirroring.

Reply:
Accepted.